docs/using.rst

```diff
@@ -50,182 +16,9 @@ document; an exhaustive list also appears near the end of the document.
-
-
-Operating System Packages
-^^^^^^^^^^^^^^^^^^^^^^^^^
-
-**FreeBSD**
-
-  * Port: ``cd /usr/ports/security/py-certbot && make install clean``
-  * Package: ``pkg install py27-certbot``
-
-**OpenBSD**
-
-  * Port: ``cd /usr/ports/security/letsencrypt/client && make install clean``
-  * Package: ``pkg_add letsencrypt``
-
-**Arch Linux**
-
-.. code-block:: shell
-
-   sudo pacman -S certbot
-
-**Debian**
-
-If you run Debian Stretch or Debian Sid, you can install certbot packages.
-
-.. code-block:: shell
-
-   sudo apt-get update
-   sudo apt-get install certbot python-certbot-apache
-
-If you don't want to use the Apache plugin, you can omit the
-``python-certbot-apache`` package.
-
-Packages exist for Debian Jessie via backports. First you'll have to follow the
-instructions at http://backports.debian.org/Instructions/ to enable the Jessie backports
-repo, if you have not already done so. Then run:
-
-.. code-block:: shell
-
-   sudo apt-get install letsencrypt python-letsencrypt-apache -t jessie-backports
-
-**Fedora**
-
-.. code-block:: shell
-
-    sudo dnf install letsencrypt
-
-**Gentoo**
-
-The official Certbot client is available in Gentoo Portage. If you
-want to use the Apache plugin, it has to be installed separately:
-
-.. code-block:: shell
-
-   emerge -av app-crypt/letsencrypt
-   emerge -av app-crypt/letsencrypt-apache
-
-When using the Apache plugin, you will run into a "cannot find a cert or key
-directive" error if you're sporting the default Gentoo ``httpd.conf``.
-You can fix this by commenting out two lines in ``/etc/apache2/httpd.conf``
-as follows:
-
-Change
-
-.. code-block:: shell
-
-   <IfDefine SSL>
-   LoadModule ssl_module modules/mod_ssl.so
-   </IfDefine>
-
-to
-
-.. code-block:: shell
-
-   #<IfDefine SSL>
-   LoadModule ssl_module modules/mod_ssl.so
-   #</IfDefine>
-
-For the time being, this is the only way for the Apache plugin to recognise
-the appropriate directives when installing the certificate.
-Note: this change is not required for the other plugins.
-
-**Other Operating Systems**
-
-OS packaging is an ongoing effort. If you'd like to package
-Certbot for your distribution of choice please have a
-look at the :doc:`packaging`.
-
-
-Installing from source
-^^^^^^^^^^^^^^^^^^^^^^
-
-Installation from source is only supported for developers and the
-whole process is described in the :doc:`contributing`.
-
-.. warning:: Please do **not** use ``python setup.py install`` or
-   ``python pip install .``. Please do **not** attempt the
-   installation commands as superuser/root and/or without virtual
-   environment, e.g. ``sudo python setup.py install``, ``sudo pip
-   install``, ``sudo ./venv/bin/...``. These modes of operation might
-   corrupt your operating system and are **not supported** by the
-   Certbot team!
-
 .. _plugins:
 
-Getting certificates (and chosing plugins)
+Getting certificates (and choosing plugins)
 ==========================================
```

Comment:
While building the docs I see:
```
docs/using.rst:22: WARNING: Title underline too short.
```
It's complaining because it wants another `=` character because you lengthened the title.